codebase/superdarn/src.lib/tk/elevation.1.0/src/elevation_v2.c

```diff
@@ -17,7 +17,7 @@
 
 /* SGS somehow need to pass in options for allowing:                         *
  *     negative elevation angles and residual phase                          */
-
+//TODP what do these parameters mean  like psi_obs? 
```

Comment:
`psi_obs` is the measured/observed ("obs") phase difference, in the range [-pi,pi). See [line 123](https://github.com/SuperDARN/rst/blob/665a554e0a037bc42f9d51ee3e1b170dc7cf90bc/codebase/superdarn/src.lib/tk/elevation.1.0/src/elevation_v2.c#L123)

This code estimates the true phase lag (many multiples of 2pi) from the observed phase difference, using the assumption that the true phase lag lies in the range (`psi_max-2pi`,`psi_max`].

`psi_max` is the maximum possible value of the phase lag. It depends on the frequency, the beam orientation, the main/auxiliary array positions, and `tdiff`

Reply:
Fixing typo in case you want to search for TODO's later
```suggestion
//TODO what do these parameters mean  like psi_obs? 
```